apps/aecore/lib/aecore/oracle/tx/oracle_response_tx.ex

```diff
@@ -94,7 +94,7 @@ defmodule Aecore.Oracle.Tx.OracleResponseTx do
     updated_interaction_objects = %{
       interaction_objects
       | response: tx.response,
-        expires: interaction_objects.expires + interaction_objects.response_ttl,
+        expires: interaction_objects.response_ttl + block_height,
```

Comment:
so response expiry is relative to current block height?

Reply:
We adjusted like epoch does. `expires` is sum of `response_ttl` and `block_height`